application/account-management/Tests/Api/Users/UserEndpointsTests.cs

```diff
@@ -74,6 +74,88 @@ public async Task GetUser_WhenInvalidUserId_ShouldReturnBadRequest()
         await EnsureErrorStatusCode(response, HttpStatusCode.BadRequest, $"""Failed to bind parameter "UserId Id" from "{invalidUserId}".""");
     }
     
+    [Fact]
+    public async Task GetUsers_WhenSearchingBasedOnUserEmail_ShouldReturnUser()
+    {
+        // Arrange
+        var searchEmail = string.Concat(DatabaseSeeder.User1ForSearching.Email.Take(4));
```

Comment:
I had to debug to understand what `.Take(4)` did :) I would do `var searchEmail = DatabaseSeeder.User1ForSearching.Email.Substring(0, 4);`

But I think this is a bit error prone. The User1ForSearching has the email `user2@search.com`. What happens if someone addes `user3@search.com`. I think it would be even more clear if we did:

`var searchEmail = "user2";` Or even gave the user a name like "John Doe", and then searched for "John"?